### PR TITLE
Project Structure + Entity Setup

### PR DESCRIPTION
**Pull request by MHassanTariq**
Changes:
- CI/CD pipeline to restrict merging into master from any branch other than develop
- Setup of Users entity
- Initialisation of DB in bootstrap

### CODE REVIEW

File: src/config/ormconfig.ts
Comment (MHassanTariq) [inline src/config/ormconfig.ts:9]: This will be reverted. 
Waiting on @MHA10 's response on #2 

I wasn't able to read these variables from .env.

**Comment by MHassanTariq** [comment]
Reminder to revert DB configs hardcoded in the `src/config/ormconfig.ts` file before merge.

File: src/config/app.config.module.ts
Comment (MHassanTariq) [inline src/config/app.config.module.ts]: Please do review this. Should this import be in config.

**Review by MHA10 — APPROVED** [review]

**Pull request merged**
